tests/base/class-aioseop-test-base.php

```diff
@@ -167,6 +167,52 @@ class AIOSEOP_Test_Base extends WP_UnitTestCase {
 		);
 	}
 
+	/**
+	 * (Barebone) Setup Posts and return IDs
+	 *
+	 * @param int   $amount
+	 * @param array $args
+	 * @return array {
+	 *     @type int $post_ID ID of the post inserted.
+	 * }
+	 */
+	protected function setup_posts_return_IDs( $amount, $args = array() ) {
```

Comment:
@EkoJR I don't understand why this method needs to be there to check if the factory has created the post type/title/content correctly. This method is just a wrapper around the create_many method and trying to verify that that factory works so I'm confused about the need for this.

Reply:
@EkoJR which problem are you referring to in that commit?

Reply:
It seems GitHub changed the URL. It's this exact URL `https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/1588/commits/08d3fbae1c33a9f84e6b0119eed1ec0338dd8509#r171503449`

Reply:
@EkoJR I think you can just add another element (elements, in case you decided to add `ids_with` and `ids_without`) to the returned array https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/master/tests/base/class-aioseop-test-base.php#L164 and add the post ids there.

Reply:
@contactashish13 I’m still unable to. I need to be able to control the content in order to test long content with descriptions, as well as varied content.
Isolated testing is also a concern. Switching functions will mix in unnecessary variables and if something changes with setup_posts (primarily with descriptions) then it could break the test function; which encapsulation would help to prevent.